paddle2onnx/op_mapper/nn.py

```diff
@@ -50,9 +50,15 @@ def opset_1(cls, graph, node, **kw):
             attrs['auto_pad'] = 'VALID'
         else:
             attrs['pads'] = pads
+        input_name = graph.get_name(node.input('Input', 0), with_remove=True)
+        # graph.make_node(
```

Comment:
remove useless codes

Reply:
done

Reply:
> @yeliang2258 Let's put a simple technical document here to describe how this works. I noticed there are some new variables and functions(name_dict, change_dict, update_graph.....), I think it's better that we add some annotations to make the pull request more reviewable.

Yes, I think this is necessary, and I will submit relevant technical documents later.

Reply:
> @yeliang2258 Let's put a simple technical document here to describe how this works. I noticed there are some new variables and functions(name_dict, change_dict, update_graph.....), I think it's better that we add some annotations to make the pull request more reviewable.

Yes, I think this is necessary, and I will submit relevant documents later.